src/pretix/base/middleware.py

```diff
@@ -249,7 +249,7 @@ def process_response(self, request, resp):
 
         h = {
             'default-src': ["{static}"],
-            'script-src': ['{static}', 'https://checkout.stripe.com', 'https://js.stripe.com'],
+            'script-src': ['{static}', 'https://checkout.stripe.com', 'https://js.stripe.com', 'https://pay.google.com'],
```

Comment:
LGTM, but I wonder… why do we even have stripe hardcoded here in a core module? :thinking:

Reply:
We can absolutely move that to the stripe signal - I'm assuming this is a leftover of when we didn't implement custom CSPs that much...

Reply:
Will do.

But good news: I think we can drop checkout.stripe.com - as this seems to be a leftover of long forgotten times, when we actually offered Stripe Checkout :)